### DISCUSSION TITLE
Registro de nombres

### DISCUSSION THREAD

**Discussion by santiemanuel** [santiemanuel/milProgramadoresQA#18] [Cursado Especial]
Buenos días, aquí pueden responder con su nombre y apellido (no hace falta correo) para que tenga la lista que debo controlar que realiza esta nueva rama en el curso. Si tienen alguna duda más la pueden dejar aquí.

**Comment by LeoChinoPuca** [comment]
Buenas tardes prof. Soy Leonardo Puca , estoy también en lo de la Upateco y voy a seguir cursando aquí también. Saludos

**Comment by Acentor** [comment]
Hola, mi nombre es Juan Artigas, yo esta semana, si se me complico, de todos modos trato de seguir la clase, ya que lo de upateco esta apenas iniciado, la semana que viene ya no tendré problemas, porque ya pedí el cambio de comisión, y quedarían los horarios acomodados

**Comment by MiriFV** [comment]
Miranda Noelia Frias Vazquez

**Comment by lucasromanh** [comment]
Lucas Román

**Reply by cmn2022** [reply]
Buenas tardes,  soy Claudia Mariela Nuñez  .
Con respecto a los coloquios prácticos,  se realizarían en  la  primera media hora de clases ?

**Reply by santiemanuel** [reply]
Eso sería lo ideal, Claudia. Lo hablaré con la profesora para que quede definido.

**Comment by santiemanuel** [comment]
Les escribo aquí para notificarles sobre el ejercicio a realizar por la clase del 20. Lo encuentran en la sección, o haciendo clic en el número #26
